From c15fc4754a05cc2b2aec4ca75f87bf3d61578843 Mon Sep 17 00:00:00 2001
From: Mike Frysinger <vapier@gentoo.org>
Date: Wed, 20 Apr 2016 00:14:21 -0400
Subject: [PATCH] include sys/sysmacros.h for major/minor/makedev

These funcs have been defined in sys/sysmacros.h forever, and Linux C
libs are moving away from implicitly including it.  Update the files
to include it explicitly.
---
 src/firejail/fs_dev.c | 1 +
 1 file changed, 1 insertion(+)

diff --git a/src/firejail/fs_dev.c b/src/firejail/fs_dev.c
index 97ee9de..2f66980 100644
--- a/src/firejail/fs_dev.c
+++ b/src/firejail/fs_dev.c
@@ -28,6 +28,7 @@
 #ifndef _BSD_SOURCE
 #define _BSD_SOURCE 
 #endif
+#include <sys/sysmacros.h>
 #include <sys/types.h>
 
 static void create_char_dev(const char *path, mode_t mode, int major, int minor) {
-- 
2.7.4

